web/packages/design/src/SlideTabs/SlideTabs.tsx

```diff
@@ -220,6 +266,47 @@ function toFullTabSpec(spec: TabSpec, index: number): FullTabSpec {
   };
 }
 
+function StatusIconOrSpinner({
+  showSpinner,
+  statusKind,
+  size,
+  color,
+  ariaLabel,
+}: {
+  showSpinner: boolean;
+  statusKind: StatusKind | undefined;
+  size: Size;
+  color: string | undefined;
+  ariaLabel: string | undefined;
+}) {
+  if (showSpinner) {
+    return <Spinner delay="none" size={size} />;
+  }
+
+  // This is one of these rare cases when there is a difference between
+  // property being undefined and not present at all: undefined props would
+  // override the default ones, but we want it them to interfere at all.
+  const optionalProps: { color?: string; 'aria-label'?: string } = {};
+  if (color != undefined) {
```

Comment:
```suggestion
  if (color !== undefined) {
```

Reply:
Is it really necessary to be so strict about it? I mean, if somehow `color` (and `ariaLabel` below) become `null`, we're gonna have a bug.